catalog.html

```diff
@@ -9,140 +9,156 @@
-              <li><a href="#">Клиенты</a></li>
-              <li class="shop"><a href="#">Магазин</a></li>
-              <li><a href="#">Контакты</a></li>
-            </ul>
-              <a class="basket" href="#">Корзина</a>
-           </nav>
-           <h1 class="catalog-preview">Магазин готовых шаблонов</h1>
-      </div>
-    </header>
-    <main>
+  <header class="main-header">
+    <div class="container">
+      <nav class="main-navigation">
+        <a class="logo" href="index.html"><img src="img/logo.svg" width="160" height="65" alt="Нёрдс"></a>
+        <ul class="navigation-list">
+          <li><a href="#">Студия</a></li>
+          <li><a href="#">Клиенты</a></li>
+          <li class="shop"><a href="#">Магазин</a></li>
+          <li><a href="#">Контакты</a></li>
+        </ul>
+          <a class="basket" href="#">Корзина</a>
+      </nav>
+          <h1 class="catalog-preview">Магазин готовых шаблонов</h1>
+    </div>
+  </header>
+  <main>
     <div class="container catalog-container">
      <section class="input-form">
-        <form action="https://echo.htmlacademy.ru/">
-            <fieldset>
-             <legend>Стоимость:</legend>
-               <div class="filter">
-                 <button></button>
-                 <button></button>
-               </div>
-               <div class="filter-interval">
-                 <label>От<input type="text" name="min-price" value="0"></label>
-                 <label>До<input type="text" name="max-price" value="15000"></label>
-               </div>
-            </fieldset>
-            <fieldset>
-             <legend>Сетка:</legend>
-               <label><input type="radio" name="grid" value="adaptive" checked>Адаптивная</label>
-               <label><input type="radio" name="grid" value="fixed">Фиксированная</label>
-               <label><input type="radio" name="grid" value="rubber">Резиновая</label>
-            </fieldset>
-            <fieldset>
-             <legend>Особенности:</legend>
-               <label><input type="checkbox" value="slider" checked>Слайдер</label>
-               <label><input type="checkbox" value="advantages">Блок преимуществ</label>
-               <label><input type="checkbox" value="news" checked>Новости</label>
-               <label><input type="checkbox" value="galery">Галерея</label>
-               <label><input type="checkbox" value="basket">Корзина</label>
-           </fieldset>
-            <button class="button button-show" type="submit">Показать</button>
-        </form>
+      <form action="https://echo.htmlacademy.ru/">
+        <fieldset>
+          <legend class="price">Стоимость:</legend>
+            <div class="range">
+              <div class="filter">
+                <div class="scale">
+                  <div class="bar"></div>
+                </div>
+                <button class="toggle toggle-min"></button>
+                <button class="toggle toggle-max"></button>
```

Comment:
```suggestion
                <button class="toggle toggle-min" type="button"></button>
                <button class="toggle toggle-max" type="button"></button>
```
По умолчанию значение равно `submit`. Если кнопка является дочерним элементом формы, то клик по такой кнопке отправляет данные. Проверьте это, кликнув по одной из этих кнопок.